addons/mail/static/src/components/channel_member/channel_member.xml

```diff
@@ -2,8 +2,8 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChannelMember" owl="1">
-        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMemberView.channelMember.partner.localId" t-ref="root">
-            <div class="o_ChannelMember_avatarContainer position-relative flex-shrink-0 o_cursor_pointer" t-on-click="channelMemberView.onClickMemberAvatar">
+        <div class="o_ChannelMember d-flex align-items-center mx-3 py-2 pl-2" title="Open Chat" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMemberView.channelMember.partner.localId" t-on-click="channelMemberView.onClickMember" t-ref="root">
```

Comment:
I believe `pl-2` should be `px-2` instead, the name can potentially go all the way to the right and overflow/ellipsis.

Reply:
Is `mx-3` still appropriate now that there is also padding? Maybe the margin could be reduced a bit to compensate?

Reply:
`my-1` has been replaced by a `py-2`, is it intended to increase the spacing between members?

Reply:
I have talked with NGR about replacing `my-1` with `py-2` to give a proper spacing between two channel members.